src/Components/People.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+class People extends Component {
+  constructor() {
+    super();
+    this.state = {
+      searchPerson: "",
+      currentPerson: {},
+      personName: "",
+      isError: false,
+    };
+  }
+
+  handleChange = (e) => {
+    this.setState({
+      searchPerson: e.target.value,
+    });
+  };
+
+  handleSubmit = async (e) => {
+    e.preventDefault();
+
+    const { searchPerson } = this.state;
+    try {
+      const { data } = await axios.get("https://ghibliapi.herokuapp.com/people");
+      const personName = data.find((person) => person.name === searchPerson);
+      if(personName === undefined) {
+        this.setState({
+          searchPerson: "",
+          currentPerson: {},
+          personName: "",
+          error: true,
+        });
+      }
+      this.setState({
```

Comment:
This line is missing the else statement. This means that if personName is undefined, it will run setState twice, once inside the if statement and again right after. Double check this logic. It can most likely be rewritten to avoid this